docs/docs/getting-started/introduction.md

```diff
@@ -16,15 +17,17 @@ import FishingPoleSVG from "@site/static/icons/fishing-pole.svg";
 import GearSVG from "@site/static/icons/gear.svg";
 import AiryBubbleSVG from "@site/static/icons/airy-bubble.svg";
 
-## What is Airy Core?
-
 <TLDR>
 
 Airy Core is an **open source**, **fully-featured**, **production-ready**
 messaging platform.
```

Comment:
could you change *messaging* to *conversational* here, we use *conversational platform* through the docs